package.json

```diff
@@ -73,50 +71,55 @@
 				"runtime": "node",
 				"configurationAttributes": {
 					"launch": {
+						"// `type`, `request` and `name` are required implicitly.": null,
 						"required": [
-							"program",
-							"args"
+							"program"
 						],
 						"properties": {
-							"commandLineArguments": {
+							"args": {
+								"// type: Array<string> -> warning `Incorrect type. Expected Array<string>` in launch.json": null,
+								"description": "Command line arguments. (i.e. [\"--opt\", \"arg\"])",
+								"default": []
+							},
+							"cwd": {
 								"type": "string",
-								"description": "Deprecated. Use `args` instead.",
-								"default": "`commandLineArguments` is deprecated. Use `args` instead."
+								"description": "Working directory. (i.e. `cd $cwd; bashdb <options> $program` will be executed internally).",
+								"default": "${workspaceFolder}"
 							},
-							"scriptPath": {
+							"program": {
 								"type": "string",
-								"description": "Deprecated. Use `program` instead.",
-								"default": "`scriptPath` is deprecated. Use `program` instead."
+								"description": "Absolute path to a script. See also: https://code.visualstudio.com/docs/editor/variables-reference",
+								"default": "${file}"
 							},
-							"args": {
+							"pathBash": {
 								"type": "string",
-								"description": "Command line arguments.",
-								"default": ""
+								"description": "Specify absolute path to bash.",
+								"default": "c:/Windows/System32/bash.exe"
```

Comment:
I wonder how to handle different systems here... Since the default value is the one that is auto-filled when adding single new entry to existing launch.json.

Reply:
Oh, and the value for windows must be "C:\\Windows\\**sysnative**\\bash.exe", not "c:\\Windows\\System32\\bash.exe".

Reply:
I think `C:\Windows\...` is enough. macOS and Linux users can easily change this value to `/usr/...`.
On the other hand, if the default value was `/usr/...`, Windows users will get errors without suspecting incorrect `pathBash`.

Reply:
Changed to `System32` unintentionally, sorry.
But `C:\Windows\System32\bash.exe` is the correct path on my 64 bit Windows. Details can be found by googling "bash system32 sysnative". We should consider how to deal with it.